lib/grape/middleware/logger.rb

```diff
@@ -64,8 +69,12 @@ def call!(env)
   end
 
   def after(status)
-    logger.info "Completed #{status} in #{((Time.now - start_time) * 1000).round(2)}ms"
-    logger.info ''
+    log_info(
```

Comment:
I'd prefer to keep this as 2 statements, instead of creating a new array object for it